### PR TITLE
Added fields to user signup and edit

### PR DESCRIPTION
**Pull request by XR89**
this be the updated sign up form

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+
+   before_action :configure_permitted_parameters, if: :devise_controller?
```
Comment (izumilink95) [inline app/controllers/application_controller.rb:3]: Excuse me, what is this monstrous extra space?
```suggestion
  before_action :configure_permitted_parameters, if: :devise_controller?
```

**Comment by Jchew1234** [comment]
Looks good to me.

**Review by a-jennings — APPROVED** [review]
All looks good to me

**Review by Jchew1234 — APPROVED** [review]
Approved

**Review by izumilink95 — APPROVED** [review]
Looking good!

**Pull request merged**